vmngclient/api/templates/models/supported.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+from vmngclient.api.templates.models.cisco_aaa_model import CiscoAAAModel
+from vmngclient.dataclasses import AlarmData
+
+
+class FeatureTemplateType(Enum):
+    cisco_aaa: str = "cedge_aaa"
+
+
+supported_models = [CiscoAAAModel, AlarmData]  # AlarmData provided just for mypy (>1 class needed, only 1 model)
```

Comment:
I think Security Template is also supported.

Reply:
added just for wip, after rebase will be fixed